package.json

```diff
@@ -43,6 +43,6 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
-    "node-html-markdown": "^1.1.3"
+    "node-html-markdown": "^1.3.0"
```

Comment:
Updated the package to the latest patch version. In the current version strikethrough delimiter wasn't exposed